sdk/core/azure-core/inc/azure/core/internal/diagnostics/log.hpp

```diff
@@ -79,8 +80,8 @@ namespace Azure { namespace Core { namespace Diagnostics { namespace _internal {
     public:
       /** @brief Configure a LogStringBuffer with the specified logging level. */
       LoggerStringBuffer(Logger::Level level) : m_level{level} {}
-      LoggerStringBuffer(LoggerStringBuffer&& that) = default;
```

Comment:
I think we can summarize the idea behind this change into one sentence and add it to the core changelog.

Reply:
Makes sense. Adding.